content/docs/languages/go/basics.md

```diff
@@ -356,21 +359,22 @@ do this for our `RouteGuide` service:
 
 ```go
 flag.Parse()
-lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
+lis, err := net.Listen("tcp", fmt.Sprintf("localhost:%d", *port))
```

Comment:
Note the added "localhost" (which comes from the example code in the repo).

Reply:
IIRC this is preferred (using localhost), and we had problems when it was omitted on some systems/setups.